src/components/Users/ManageUsers.tsx

```diff
@@ -115,7 +115,7 @@ export default function ManageUsers() {
         activeTab={activeTab}
         onTabChange={setActiveTab}
       />
-      <Pagination totalCount={userListData.count} />
+      <Pagination totalCount={userListData?.count ?? 0} />
```

Comment:
this is a hack :) 

we shouldn't be showing the pagination component when it's loading. hence the type issues. not needed in the first place right?

Reply:
The Pagination component is part of `manageUsers`. If manageUsers is in a loading state `<Loading />`, we ensure that the Pagination component isn’t displayed.